### PR TITLE
fix content type open close

### PR DESCRIPTION
**Pull request by lindenmckenzie**
### What

Fixed markup for collapsible to match design-system as was not opening and closing. 

### How to review

Boot up frontend-search-controller, dp-design-system
Tunnel to API Router
Make search request
Try to open / close content-type

### Who can review

FE dev.

### CODE REVIEW

**Comment by lindenmckenzie** [comment]
Much of the change here is indentation - will highlight actual changes.

File: assets/templates/partials/filter.tmpl [hunk 1]
```diff
@@ -67,70 +67,68 @@
     <input type="hidden" name="q" value="{{ $query }}"/>
     <input type="hidden" name="sort" value="{{$sort}}"/>
     <fieldset class="ons-fieldset">
-        <div id="collapsibleContentType" class="ons-collapsible ons-js-collapsible" data-btn-close="Hide this" data-open="true">
```
Comment (lindenmckenzie) [inline assets/templates/partials/filter.tmpl:70]: Changed to `<details>`

File: assets/templates/partials/filter.tmpl [hunk 2]
```diff
@@ -67,70 +67,68 @@
...
-        <div class="ons-collapsible__heading ons-js-collapsible-heading">
```
Comment (lindenmckenzie) [inline assets/templates/partials/filter.tmpl:71]: Changed to `<summary>`

File: assets/templates/partials/filter.tmpl [hunk 3]
```diff
@@ -67,70 +67,68 @@
...
-            <div class="ons-collapsible__controls">
```
Comment (lindenmckenzie) [inline assets/templates/partials/filter.tmpl:72]: Removed this `div`

**Comment by lindenmckenzie** [comment]
N.B. `collapsible` has been removed in a later version of design-system and been replaced by `details` - we maybe should consider moving this to the accordion component in future.

**Review by nshumoogum — APPROVED** [review]
Tested locally and content type drop down now closes and opens

**Pull request merged**